### PR TITLE
Lifinity protocol constant msol holder parsing

### PR DESCRIPTION
**Pull request by ochaloup**
Lifinity protocol mSOL holder parsing in constant way as it was discussed with Cerba and Lifinity guys.
(there could be a change in owner of the UXD pool but Durden will provide that info later)

### CODE REVIEW

File: src/snapshot/parser.service.ts [hunk 1]
```diff
@@ -632,6 +634,63 @@ export class ParserService {
...
+    return result;
+  }
+
+  private lifinity(db: SQLite.Database): Record<string, BN> {
+    this.logger.log('Parsing Lifinity');
+    const buf: Record<string, BN> = {};
+    const lifinityVaults = this.getLifinityVaultsAndOwners();
+    for (const { msolVault, owner } of lifinityVaults) {
+      const vaultAmount = this.getTokenAccountBalance(db, msolVault);
+      console.log('vault', msolVault, 'vaultAmount', vaultAmount?.toString());
```
Comment (butonium) [inline src/snapshot/parser.service.ts:681]: console.log?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (butonium) [inline .vscode/settings.json:1]: I think this should be in .gitignore

File: src/snapshot/parser.service.ts [hunk 3]
```diff
@@ -632,6 +634,63 @@ export class ParserService {
     return buf;
   }
 
+  private getLifinityVaultsAndOwners() {
```
Comment (butonium) [inline src/snapshot/parser.service.ts:637]: so we treat lifinity as holder, right? (send them the rewards)

**Review by butonium — CHANGES_REQUESTED** [review]

File: src/snapshot/parser.service.ts (hunk 3 above)
Comment (ochaloup) [reply src/snapshot/parser.service.ts:637]: yes, the lifinity should be considered as the holder, i.e., sending them rewards. they are able to sign the transactions with the treasury account as mentioned here

File: .vscode/settings.json (hunk 2 above)
Comment (ochaloup) [reply .vscode/settings.json:1]: This is interesting as the .gitignore considers the `settings.json` to not be ignored. It seems that intention of the person who started this project was to consider settings.json as part of the git repo.
Anyway, removing my changes.

**Comment by ochaloup** [comment]
@butonium I managed to update this easy one PR review, please take a look.
For the other ones I need to manage later (I will ping you on discord).

**Review by butonium — APPROVED** [review]

**Pull request merged**